src/tasks/WasmAppBuilder/WasmAppBuilder.cs

```diff
@@ -13,38 +13,16 @@
 using Microsoft.Build.Framework;
 using Microsoft.Build.Utilities;
 
-public class WasmAppBuilder : Task
-{
-    [NotNull]
-    [Required]
-    public string? AppDir { get; set; }
+namespace Microsoft.WebAssembly.Build.Tasks;
```

Comment:
Shouldn't there be a need to update task registration in `WasmApp.targets`?

Reply:
It can be either the type name, or the full name. But it is a good practice to use the full name. I will update that in a follow up PR.

Reply:
Updated in this PR itself.

Reply:
> It can be either the type name, or the full name. But it is a good practice to use the full name. I will update that in a follow up PR.

Thanks, I did not know that.